atlas_admin/__init__.py

```diff
@@ -59,7 +60,25 @@ def instances():
         summaryInstances['cost_multiplier'] = cost_multiplier
         summaryInstances['cost'] = ((xs[0] * xs[1]) + (s[0] * s[1]) + (m[0] * m[1]) +
                                     (l[0] * l[1]) + (xl[0] * xl[1]) + (e[0] * e[1]))
-    return render_template('instances/summary.html', summaryInstances=summaryInstances)
+    return render_template(
+        'instances/summary.html',
+        summaryInstances=summaryInstances,
+        statBreakdown=statBreakdown)
+
+
+@atlas_admin.route('/instances/stats')
+def theme_summary():
```

Comment:
I don't think we need this function and the related template, they replicate what is on the summary page

Reply:
done